src/components/tag-list/tag-list.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import styles from './tag-list.css';
+import { useRef } from 'react';
+import { useEffect } from 'react';
+
+export default function TagList(props) {
+  const [active, setActive] = useState(false);
+  const [focusIndex, setFocusIndex] = useState(0);
```

Comment:
Rename to `activeIndex`

We want to keep terminology consistent with accessibility:

- Something can be default focusable:
  - It could be focused
  - One of its children has a tabIndex of 0
- Something can be active:
  - One of its children is currently focused (or was focused)
  - One of its children has a tab index of 0
  - The moment a widget is active, it is no longer default focusable because its active state is now managed!
- Something can be inactive:
  - All of its children have a tab index of -1